### ISSUE TITLE
Unable to replicate the performance of your re-implemented GenLaneNet

### ISSUE THREAD

**Issue by RunkaiZhao** [YaoChengTang/Sparse-Point-Guided-3D-Lane-Detection#4]
Great work! Thank you for your contribution to the community.

I was impressed that your re-implementation of GenLaneNet achieved a 42.8% F1 score, as mentioned in Table 2 of your paper. However, when I attempted to train the GenLaneNet using your code, the F1 score remained at zero on the validation set. Could you please provide some explanations or suggestions? Thank you in advance.

**Comment by YaoChengTang** [comment]
Following the [training procedure of GenLaneNet](https://github.com/yuliangguo/Pytorch_Generalized_3D_Lane_Detection?tab=readme-ov-file), we first train the segmentation subnetwork and then train the 3D-geometry subnetwork. No more complex strategies or tricks are used. Please provide your training details for discussion.
